src/main/java/org/ideoholic/curium/model/feescategory/action/FeesAction.java

```diff
@@ -90,8 +91,7 @@ public String feesStructure() {
 
 	@PostMapping("/deleteFeesCategory")
 	public String deleteFeesCategory() {
-		String studentId = feesActionAdapter.deleteFeesCategory();
```

Comment:
Why was this line removed?
`String studentId = feesActionAdapter.deleteFeesCategory();`